### PR TITLE
[community] Added Member Profile: Tejas

### LINKED ISSUE #4418: [Community] Member Profile: Tejas
**Current Behavior**

@0xt3j4s has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @0xt3j4s as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @0xt3j4s
- Twitter: [tejas_jamdade04](https://twitter.com/tejas_jamdade04)
- LinkedIn: [0xt3j4s](https://www.linkedin.com/in/0xt3j4s/)
- Meshery Cloud: [60d7dbbf-2489-4f41-81b9-8edf4605d939](https://meshery.layer5.io/user/60d7dbbf-2489-4f41-81b9-8edf4605d939)
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sambhavgupta0705**
**Description**

This PR fixes #4418 

**Notes for Reviewers**
Added the member profile of Tejas
@0xt3j4s 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Nice work @sambhavgupta0705 💪  
Looks good to merge from my side ✅

File: src/collections/members/tejas-jamdade/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+position: contributor
+image: ./tejas-jamdade.webp
+github: 0xt3j4s
+twitter: tejas_jamdade04
+linkedin: 0xt3j4s
+location: Pune, Maharashtra, India
+bio: Hey there. I am Tejas, an undergraduate student from the Indian Institute of Technology, Jodhpur. I am passionate about Software Development, Open Source, DevOps and Security. I am a hands-on learner and prefer exploring and learning new technologies by building cool projects.
+badges:
+    - meshery
+    - meshmap
```
Comment (Chadha93) [inline src/collections/members/tejas-jamdade/index.mdx:12]:
```suggestion
```

**Comment by Ghat0tkach** [comment]
@sambhavgupta0705 please sign your commits, refer to this [link](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits) .
Thanks for your contribution 🎉

**Comment by sambhavgupta0705** [comment]
@Ghat0tkach I tried but unable to signoff the commit

**Comment by l5io** [comment]
🚀 Preview for commit 7dd867244bbc0bd337e5487ba9442380019ede8d at: https://6495e52749d2ee6a2995c55c--layer5.netlify.app

**Review by Saksham294 — COMMENTED** [review]
Hey @sambhavgupta0705 I think the mdx file is missing three dashes (---) in the end. Kindly check.

**Comment by l5io** [comment]
🚀 Preview for commit 6688bf8add1bce256aaece1b8ff46a57b8a9010f at: https://6495ea157184ed6d0392777c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 68f95030f9790794c1d4cf589d953faaea7b454b at: https://6495ee92bd896d6f3d91537f--layer5.netlify.app

**Comment by Saksham294** [comment]
LGTM 🚀

**Comment by leecalcote** [comment]
@0xt3j4s yay! 🎈

**Review by leecalcote — APPROVED** [review]

**Review by 0xt3j4s — APPROVED** [review]
LGTM!

**Review by DarrenDsouza7273 — APPROVED** [review]
Good work @sambhavgupta0705

**Comment by l5io** [comment]
🚀 Preview for commit aed4a9aca70bfb87992040c89f843f7c9f636849 at: https://64968087f4f57426549b5966--layer5.netlify.app

**Review by Saksham294 — APPROVED** [review]

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**